themes/steeef.zsh-theme

```diff
@@ -10,7 +10,7 @@
 export VIRTUAL_ENV_DISABLE_PROMPT=1
 
 function virtualenv_info {
-    [ $VIRTUAL_ENV ] && echo '('$fg[blue]`basename $VIRTUAL_ENV`%{$reset_color%}') '
+    [ $VIRTUAL_ENV ] && echo '('%F{blue}`basename $VIRTUAL_ENV`%f') '
```

Comment:
One other fix here: use `${VIRTUAL_ENV:h}` instead of the call to `basename`.
Also `[ $VIRTUAL_ENV ]` could be switch for `[[ -n "$VIRTUAL_ENV" ]]`.